pretyping/nativenorm.ml

```diff
@@ -469,11 +469,19 @@ let start_profiler_linux profile_fn =
       [|perf; "record"; "-g"; "-o"; profile_fn; "-p"; string_of_int coq_pid |]
       Unix.stdin dev_null dev_null
   in
-  (* doesn't seem to be a way to test whether process creation succeeded *)
+  (* doesn't seem to be a way to test whether process creation succeeded
```

Comment:
Actually there is a way that works across versions, but you need to wait for the process and check if exit code is 127